docs/literate/src/files/first_steps/getting_started.jl

```diff
@@ -0,0 +1,250 @@
+
+# To edit an existing elixir, you first have to find a suitable one and then copy it to a local
+# folder. Let's have a look at how to download the `elixir_euler_ec.jl` elixir used in the previous
+# section from the [Trixi.jl GitHub repository](https://github.com/trixi-framework/Trixi.jl).
+
+# - All examples are located inside
+#   the [`examples`](https://github.com/trixi-framework/Trixi.jl/tree/main/examples) folder.
+# - Navigate to the
+#   file [`elixir_euler_ec.jl`](https://github.com/trixi-framework/Trixi.jl/blob/main/examples/tree_2d_dgsem/elixir_euler_ec.jl).
+# - Right-click the `Raw` button on the right side of the webpage and choose `Save as...`
+#   (or `Save Link As...`).
+# - Choose a folder and save the file.
+
+
+# ### Modifying an existing setup
+
+# As an example, we will change the initial condition for calculations that occur in
+# `elixir_euler_ec.jl`. In this example we consider the compressible Euler equations in two spatial
+# dimensions,
+# ```math
+# \frac{\partial}{\partial t}
+# \begin{pmatrix}
+# \rho \\ \rho v_1 \\ \rho v_2 \\ \rho e
+# \end{pmatrix}
+# +
+# \frac{\partial}{\partial x}
+# \begin{pmatrix}
+# \rho v_1 \\ \rho v_1^2 + p \\ \rho v_1 v_2 \\ (\rho e +p) v_1
+# \end{pmatrix}
+# +
+# \frac{\partial}{\partial y}
+# \begin{pmatrix}
+# \rho v_2 \\ \rho v_1 v_2 \\ \rho v_2^2 + p \\ (\rho e +p) v_2
+# \end{pmatrix}
+# =
+# \begin{pmatrix}
+# 0 \\ 0 \\ 0 \\ 0
+# \end{pmatrix},
+# ```
+# for an ideal gas with the specific heat ratio ``\gamma``.
+# Here, ``\rho`` is the density, ``v_1`` and ``v_2`` are the velocities, ``e`` is the specific
+# total energy, and
+# ```math
+# p = (\gamma - 1) \left( \rho e - \frac{1}{2} \rho (v_1^2+v_2^2) \right)
+# ```
+# is the pressure.
+# Initial conditions consist of initial values for ``\rho``, ``\rho v_1``,
+# ``\rho v_2`` and ``\rho e``.
+# One of the common initial conditions for the compressible Euler equations is a simple density
+# wave. Let's implement it.
+
+# - Open the downloaded file `elixir_euler_ec.jl` with a text editor.
+# - Go to the line with the following code:
+#   ```julia
+#   initial_condition = initial_condition_weak_blast_wave
+#   ```
+#   Here, [`initial_condition_weak_blast_wave`](@ref) is used as the initial condition.
+# - Comment out the line using the `#` symbol:
+#   ```julia
+#   # initial_condition = initial_condition_weak_blast_wave
+#   ```
+# - Now you can create your own initial conditions. Add the following code after the
+#   commented line:
+
+    function initial_condition_density_waves(x, t, equations::CompressibleEulerEquations2D)
+        v1 = 0.1 # velocity along x-axis
+        v2 = 0.2 # velocity along y-axis
+        rho = 1.0 + 0.98 * sinpi(sum(x) - t * (v1 + v2)) # density wave profile
+        p = 20 # pressure
+        rho_e = p / (equations.gamma - 1) + 1/2 * rho * (v1^2 + v2^2)
+        return SVector(rho, rho*v1, rho*v2, rho_e)
+    end
+    initial_condition = initial_condition_density_waves
```

Comment:
```suggestion
initial_condition = initial_condition_density_waves
```
Same for the lines above. Otherwise, there is a strange looking inundation.